app/models/race_edition.rb

```diff
@@ -1,12 +1,15 @@
 class RaceEdition < ActiveRecord::Base
-   belongs_to :race
-   has_many :race_entries
-
-   validates :race_id, presence: true
-   validates :date, presence: true
-   
-   def name
-      race.name + ':' + date.to_s
-   end
-   
- end
\ No newline at end of file
+  belongs_to :race
+  has_many :race_entries
+  has_many :racers, through: :race_entries
+
```

Comment:
This `has_many: through` relationship is very convenient as it allows us to deal with racers by calling `race_edition.racers` and letting ActiveRecord create the multi-join query for us.